server/integration_tests/nl_test.py

```diff
@@ -247,6 +248,14 @@ def test_demo_usa_map_types(self):
             'how many people are unemployed in zip codes of washington?'
         ])
 
+  def test_translate(self):
```

Comment:
Can you add this to explore_test.py?  We should have 2 tests: one for detect/ and another for detect-and-fulfill/

Reply:
Done.